components/script.tsx

```diff
@@ -99,9 +101,11 @@ const Script: React.FC<ScriptProps> = ({ className, messagesHeight = 'h-full', e
 
     return (
         <div className={`h-full w-full ${className}`}>
+            {showAssistantName && <h1
```

Comment:
Added the title where chat can have assitant name on the top

<img width="1144" alt="image" src="https://github.com/user-attachments/assets/e6f64589-e8f5-40ba-9c95-666e299a09c8">